spec/models/profile_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Profile do
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:cell_phone) }
+    it { is_expected.to validate_numericality_of(:weight).is_greater_than(0) }
+    it { is_expected.to validate_numericality_of(:height).is_greater_than(0) }
+    it { is_expected.to belong_to :user }
+  end
+
+  describe "birthday" do
+    subject(:profile) { described_class.new }
+
+    it { is_expected.to validate_presence_of(:birthday) }
+
+    it "is valid with past birthday" do
+      profile.birthday = Time.zone.today - 1
+      profile.valid?
+      expect(profile.errors.keys).not_to include :birthday
+    end
+
+    it "is valid with current birthday" do
+      profile.birthday = Time.zone.today
+      profile.valid?
+      expect(profile.errors.keys).not_to include :birthday
+    end
+
+    it "is invalid with future birthday" do
+      profile.birthday = Time.zone.today + 1
+      profile.valid?
+      expect(profile.errors.keys).to include :birthday
+    end
```

Comment:
Nesse caso, poderia colocar os contexts também, igual fez no validator, assim não precisa sempre fazer igual o setup do teste. Veja que você repete sempre dentro do example:
```ruby
      profile.birthday = date
      profile.valid?
```

Outra coisa, tem o describe de "validations" isso poderia estar la dentro, o `it { is_expected.to validate_presence_of(:birthday) }` pode ficar junto dos outros, ai você só coloca contextos dentro de validations falando "when birthday is in the past", "when birthday is today", "when birthday is in the future", fica mais organizado, sabendo que isso faz parte dos testes de validação.

Reply:
Fiz como você orientou, peço que valide se as alterações estão de acordo